src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const part1 = `name: ${this.name}, `;
+
+      const part2 = `chip version: ${this.version}, wheels: ${this.wheels}`;
+
+      return part1 + part2;
```

Comment:
you can just use destructuring here
```suggestion
    get info() {
      const { name, version, wheels } = this;

      return `${name}, chip version: ${version}, wheels: ${wheels}`;
```